.buildkite/env-scripts/env.sh

```diff
@@ -1,19 +1,53 @@
 #!/usr/bin/env bash
 
-SETUP_GVM_VERSION="v0.5.1"
+source .buildkite/env-scripts/util.sh
+
+DOCS_CHANGESET="^.*\.(asciidoc|md)$
+deploy/kubernetes/.*-kubernetes.yaml"
+PACKAGING_CHANGESET="^dev-tools/packaging/
+^.go-version"
+
+REPO="beats"
 WORKSPACE="$(pwd)"
 BIN="${WORKSPACE}/bin"
 HW_TYPE="$(uname -m)"
 PLATFORM_TYPE="$(uname)"
-REPO="beats"
 TMP_FOLDER="tmp.${REPO}"
+SNAPSHOT="true"
+PYTEST_ADDOPTS=""
+OSS_MODULE_PATTERN="^[a-z0-9]+beat\\/module\\/([^\\/]+)\\/.*"
+XPACK_MODULE_PATTERN="^x-pack\\/[a-z0-9]+beat\\/module\\/([^\\/]+)\\/.*"
+
+SETUP_GVM_VERSION="v0.5.1"
+ASDF_MAGE_VERSION="1.14.0"
+SETUP_WIN_PYTHON_VERSION="3.11.0"
```

Comment:
This could be removed once we will switch to Beats win images. I thought it would be better to have win image change in a separate PR, since the current one is already overwhelmed with the changes. WDYT?

Reply:
if things work in the current hybrid state i.e. some tests using the vanilla images and some others the custom ones, I am good for us to address in the near future.